tests/profiling/collector/test_stack.py

```diff
@@ -178,37 +172,40 @@ def _fib(n):
         return _fib(n - 1) + _fib(n - 2)
 
 
-@pytest.mark.skipif(not TESTING_GEVENT, reason="Not testing gevent")
-def test_collect_gevent_thread_task():
-    r = recorder.Recorder()
-    s = stack.StackCollector(r)
-
-    # Start some (green)threads
-
-    def _dofib():
-        for _ in range(10):
-            # spend some time in CPU so the profiler can catch something
-            _fib(28)
-            # Just make sure gevent switches threads/greenlets
-            time.sleep(0)
-
-    threads = []
-    with s:
-        for i in range(10):
-            t = threading.Thread(target=_dofib, name="TestThread %d" % i)
-            t.start()
-            threads.append(t)
-        for t in threads:
-            t.join()
-
-    for event in r.events[stack_event.StackSampleEvent]:
-        if event.thread_name == "MainThread" and event.task_id in {thread.ident for thread in threads}:
-            assert event.task_name.startswith("TestThread ")
-            # This test is not uber-reliable as it has timing issue, therefore if we find one of our TestThread with the
-            # correct info, we're happy enough to stop here.
-            break
-    else:
-        pytest.fail("No gevent thread found")
+# TODO: This test assumes that it is running with a gevent-patched threading
+# module. Needs to be adapted to work again now.
+# @pytest.mark.skipif(not TESTING_GEVENT, reason="Not testing gevent")
+# def test_collect_gevent_thread_task():
+#     r = recorder.Recorder()
+#     s = stack.StackCollector(r)
+
+#     # Start some (green)threads
+
+#     def _dofib():
+#         for _ in range(10):
+#             # spend some time in CPU so the profiler can catch something
+#             _fib(28)
+#             # Just make sure gevent switches threads/greenlets
+#             time.sleep(0)
+
+#     threads = []
+#     with s:
+#         for i in range(10):
+#             t = threading.Thread(target=_dofib, name="TestThread %d" % i)
+#             t.start()
+#             threads.append(t)
+#         for t in threads:
+#             t.join()
+
+#     for event in r.events[stack_event.StackSampleEvent]:
+#         if event.thread_name == "MainThread" and event.task_id in {thread.ident for thread in threads}:
+#             assert event.task_name.startswith("TestThread ")
+#             # This test is not uber-reliable as it has timing issue, therefore
+#             # if we find one of our TestThread with the correct info, we're
+#             # happy enough to stop here.
+#             break
+#     else:
+#         pytest.fail("No gevent thread found")
```

Comment:
Do we need this test case anymore? Since we don't have `gevent` threads anymore on ddtrace land 😄